### PR TITLE
profile is not visible

### PR DESCRIPTION
**Pull request by PuneetDabral**
Signed-off-by: Puneet Dabral <puneetdabral2000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by PuneetDabral** [comment]
sir please check and merge it so i  can Calibrate my contribution ......also feel motivated ....!!

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
I dont think this would work, have you tested out the changes locally? you'd probably have to do `maintainer: yes`. Refer to the community member profile template: https://github.com/layer5io/layer5/blob/master/src/collections/members/_member-profile-template/index.mdx

**Comment by l5io** [comment]
🚀 Preview for commit 6d190e464e82aa1c2cc4f2fa28b6982169626143 at: https://613df6d1e920e028ec8449d7--layer5ng.netlify.app

**Comment by PuneetDabral** [comment]
can you please help me sir .. what i cna do

File: src/collections/members/rudraksh-pareek/index.mdx [hunk 1]
```diff
@@ -10,7 +10,8 @@ bio: I'm an undergrad from India studying Computer Science and Engineering. I li
 badges:
   - meshery
   - landscape
-  - smp
+
```
Comment (warunicorn19) [inline src/collections/members/rudraksh-pareek/index.mdx:13]:
```suggestion
  -smp
```

**Comment by PuneetDabral** [comment]
@warunicorn19 sir please check and merge it ...issue is solved

**Comment by warunicorn19** [comment]
Yes @PuneetDabral almost there, you seem to have removed the `-smp` by mistake if you could add that and then commit again, and this time remember to sign-off like you did the first time. Good work.

**Comment by PuneetDabral** [comment]
@warunicorn19 now it is ok sir ...check it and merge it

**Review by debo19 — APPROVED** [review]
@PuneetDabral code looks fine now, just sign off your commits for it to be merged. Follow these steps https://github.com/layer5io/layer5/pull/2176/checks?check_run_id=3589333190 and we'll be good to go.

**Comment by PuneetDabral** [comment]
now sir....

**Review by Tanuj22 — APPROVED** [review]
Thank you @PuneetDabral

**Comment by PuneetDabral** [comment]
sir how my name and image shows in Contributor List of this  website

**Comment by leecalcote** [comment]
@PuneetDabral you didn't list the Issue # that this PR addresses in the Issue Description.

**Comment by leecalcote** [comment]
> sir how my name and image shows in Contributor List of this website

Eventually, here - https://github.com/layer5io/layer5/graphs/contributors

**Pull request merged**